subset/switches/src/main/java/switchtest/SwitchInterrogator.java

```diff
@@ -45,24 +45,14 @@
   };
 
   String[] command = {
-    "enable",
-    "show interface port1.0.",
-    "show platform port port1.0.",
-    "show run",
-    "show stack"
+    "enable", "show interface port1.0.", "show platform port port1.0.", "show run", "show stack"
```

Comment:
Are these auto-format updates, or are you manually tweaking them?   They should be the auto-formatter, whatever that does, even if they're subjectively less readable.  All bow to the power of the auto-formatter.

Reply:
this is the auto-formatter in action. I think I corrected before. will leave as you suggest.